x-pack/plugins/enterprise_search/public/applications/enterprise_search_overview/components/product_selector/ingestion_selector.tsx

```diff
@@ -37,6 +38,7 @@ import { HttpLogic } from '../../../shared/http/http_logic';
 
 import { ConnectorIcon } from '../../../shared/icons/connector';
 import { CrawlerIcon } from '../../../shared/icons/crawler';
+import { GithubIcon } from '../../../shared/icons/github_icon';
```

Comment:
Well it is just a duplication, but looks like we already break a lot of rules previously 😬 It can stay. If you want to try already existing one

This file has the export for the same svg `x-pack/plugins/enterprise_search/public/assets/client_libraries/index.ts`